api/Event.json

```diff
@@ -1321,6 +1390,10 @@
               "version_added": "49",
               "notes": "Starting with Chrome 49, Firefox 54 and Opera 36, this property returns <a href='https://developer.mozilla.org/docs/Web/API/DOMHighResTimeStamp'><code>DOMHighResTimeStamp</code></a> instead of <a href='https://developer.mozilla.org/docs/Web/API/DOMTimeStamp'><code>DOMTimeStamp</code></a>."
             },
+            "deno": {
+              "version_added": "1.0",
+              "notes": "In Deno this represents the time since UNIX epoch, not since startup. See https://github.com/denoland/deno/issues/10835."
```

Comment:
```suggestion
              "notes": "In Deno this represents the time since UNIX epoch, not since startup. See <a href='https://github.com/denoland/deno/issues/10835'>#10835</a>."
```